Database/Corrections/sodQuestFixes.lua

```diff
@@ -296,6 +296,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.preQuestSingle] = {78506},
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [78561] = {
+            [questKeys.preQuestSingle] = {78537},
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
```

Comment:
The scraper does not handle "Provided Item:" from wowhead yet, so we manually need to remove this objective.

```suggestion
        [78561] = {
            [questKeys.objectives] = {},
            [questKeys.preQuestSingle] = {78537},
            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
        },
```

Reply:
Okay. I'll have to bear that in mind in the future :)

Fixed in: https://github.com/Questie/Questie/pull/5372/commits/bdea28a71cfdbc7f54879121a7c3f040a2a46326